CONTRIBUTING.rst

```diff
@@ -67,11 +67,10 @@ With respect to code standards, we follow `PEP8`_ and the `Google Style Guide`_.
 .. _`Google Style Guide` : https://google.github.io/styleguide/pyguide.html
 
 
-Licensing
+Licensing of contributions
```

Comment:
@dmasad -- Do you prefer "Licensing" or "Licensing of contributions" ... I kind of prefer the latter to be explicit, but I am open .